libexec/bats-core/bats-exec-test

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env bash
 set -eET
 
+# Variables used in other scripts.
 BATS_COUNT_ONLY=''
 BATS_TEST_FILTER=''
 BATS_EXTENDED_SYNTAX=''
 
 while [[ "$#" -ne 0 ]]; do
   case "$1" in
   -c)
+    # shellcheck disable=SC2034
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2034

Leaving this unused variable for backwards compatibility.